ua-snap/ardac-explorer
# Add lat/lon input component and example item with dynamic API links

This PR adds a new `LatLngSelector` component that was adapted from our Arctic-EDS version of the component, updated as needed to work with Nuxt 3 + Pinia + TypeScript.

The component uses the same parse-dms NPM module that Arctic-EDS does, so it accepts lat/lon coordinate strings in all standard formats. It also performs the same BBOX validation that Arctic-EDS does and warns the user if they have entered invalid coordinates.

I've also added a new item, "Projected permafrost and mean annual ground temperature", that uses the lat/lon component to generate CSV and JSON links based on the coordinates that were entered. This item also includes the same permafrost source data link and citation information as Arctic-EDS.

To test:
- `npm install`
- `npm run dev`
- Find the "Projected permafrost and mean annual ground temperature" item and click "Read more"
- Interact with the lat/lon widget and confirm that it works as expected with a variety of different lat/lon points and coordinate formats (e.g., `65, -147`, `64.8401° N, 147.7200° W`).
- Make sure the CSV and JSON links work as expected
